modules/multi-arch-uninstalling-using-cli.adoc

```diff
@@ -0,0 +1,52 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/multiarch-tuning-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="multi-architecture-uninstalling-using-cli_{context}"]
+
+= Uninstalling Multiarch Tuning Operator by using the CLI
+
```

Comment:
Can we add a warning here to highlight the fact that the users should remove the operand first and then remove the operator?

Reply:
Added it as a prereq.